openedx_tagging/core/tagging/api.py

```diff
@@ -136,3 +136,37 @@ def tag_object(
     Preserves existing (valid) tags, adds new (valid) tags, and removes omitted (or invalid) tags.
     """
     return taxonomy.cast().tag_object(tags, object_id)
+
+
+def autocomplete_tags(
+    taxonomy: Taxonomy,
+    search: str,
+    object_id: str = None,
+) -> QuerySet:
```

Comment:
I'm curious to know if it's possible to use `django-stubs` [`ValuesQuerySet[...]`](https://github.com/typeddjango/django-stubs/blob/e70ec130913c499e31a0ba9dbb3ae5bc33ab9eff/django-stubs/db/models/manager.pyi#L99) to define this return type more precisely, but it's not an important thing to investigate now if you're short on time.

Reply:
`ValuesQuerySet` is [imported](https://github.com/typeddjango/django-stubs/blob/e70ec130913c499e31a0ba9dbb3ae5bc33ab9eff/django-stubs/db/models/manager.pyi#L10) from `django_stubs_ext`, and needs one more dependency and [extra configuration](https://github.com/typeddjango/django-stubs/tree/master/django_stubs_ext). I think for now it would be a lot to get that type

Reply:
OK. Thanks for checking into it!